catalogue/webapp/pages/item.js

```diff
@@ -5,13 +5,149 @@ import fetch from 'isomorphic-unfetch';
 import { type IIIFManifest, type IIIFCanvas } from '@weco/common/model/iiif';
 import { iiifImageTemplate } from '@weco/common/utils/convert-image-uri';
 import { itemUrl, workUrl } from '@weco/common/services/catalogue/urls';
-import Layout12 from '@weco/common/views/components/Layout12/Layout12';
 import PageLayout from '@weco/common/views/components/PageLayout/PageLayout';
 import Paginator, {
   type PaginatorRenderFunctionProps,
 } from '@weco/common/views/components/RenderlessPaginator/RenderlessPaginator';
 import Control from '@weco/common/views/components/Buttons/Control/Control';
 import { classNames, spacing, font } from '@weco/common/utils/classnames';
+import styled from 'styled-components';
+import Layout12 from '@weco/common/views/components/Layout12/Layout12';
+
+const IIIFViewerPaginatorButtons = styled.div.attrs({
```

Comment:
Give how many `isThumbs` conditions there are in here, is it worth having `IIIFViewerThumbsButtons` & `IIIFViewerMainButtons` or something?